src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={cn({
+                'is-active': selectedTab.id === tab.id,
```

Comment:
will be better create separated variable like `isSelectedTab` and reuse

Reply:
I disagree with this, why do we need to create a separate variable for the sake of one comparison that is used only once?

Reply:
I understand why we need to use the approach you have suggested. The basic rule is "don't repeat yourself" and reuse code. But in this task, it is not necessary, because it is used only once. And creating an additional variable will be meaningless.